apps/docs/src/config.ts

```diff
@@ -50,6 +53,7 @@ export const SIDEBAR: Sidebar = {
       { text: "Paragraph", link: "06-paragraph" },
       { text: "Hero", link: "07-hero" },
       { text: "Kicker", link: "12-kicker" },
+      { text: "Icon Button", link: "17-icon-button" },
```

Comment:
gj! I think we can get rid of those numbers in links. they're not necessary and make a bit of mess when working simultaneously